### PR TITLE
docs: Update README

### LINKED ISSUE #51: Replace stock README with content specific to this site.

### PR DESCRIPTION
**Pull request by spmcbride1201**
Resolves #51

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,98 +1,50 @@
...
+category: "introduction"
+type: "lesson"
+tags:
+    - programming
+    - fabric
+    - Grey Matter
+---
+```
+
+Admittedly, most of these tags are useful for our purposes. The key exceptions to this are `title`, `lesson`, and `chapter`, which are used as follows:
```
Comment (joshua-rutherford) [inline README.md]: I think you mean "are *NOT* useful".  It reads a bit backwards if that was not your intent.

File: README.md [hunk 2]
```diff
@@ -1,98 +1,50 @@
...
+
+* chapter: An integer used to group posts together under a common header. The integer represents an index in the toCChapters array defined in /data/SiteConfig.js`. Currently, this is as follows:
```
Comment (joshua-rutherford) [inline README.md]: You have a closing *tick* after `/data/SiteConfig.js` but no opening *tick*.

File: README.md [hunk 3]
```diff
@@ -1,98 +1,50 @@
...
+
+Use `npm run lint:md` to lint your Markdown for errors
+use `npm run write-good` to check your posts for grammatical issues. 
+
+Note: we currently do not have a spellchecker
+
+## To Publish Content to GitHub Pages
+
+Run `npm run deploy` to build and deploy the latest site.
+
```
Comment (joshua-rutherford) [inline README.md]: Can we add a quick second on`npm run develop` so people know how to view it locally too?

File: README.md (hunk 1 above)
Comment (spmcbride1201) [reply README.md]: Yes, that's pretty nonsensical otherwise.

**Pull request merged**